panda/src/audiotraits/config_openalAudio.h

```diff
@@ -27,4 +27,6 @@ NotifyCategoryDecl(openalAudio, EXPCL_OPENAL_AUDIO, EXPTP_OPENAL_AUDIO);
 extern EXPCL_OPENAL_AUDIO void init_libOpenALAudio();
 extern "C" EXPCL_OPENAL_AUDIO Create_AudioManager_proc *get_audio_manager_func_openal_audio();
 
+extern ConfigVariableString device_cfg;
```

Comment:
Config variables should be named consistently, in this case `openal_device` instead of `device_cfg`.